libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -497,6 +497,33 @@ class BreezSDKMapper {
             "lastRedeemError": swapInfo.lastRedeemError
         ]
     }
+
+    static func dictionaryOf(reverseSwapPairInfo: ReverseSwapPairInfo) -> [String: Any?] {
+        return [
+            "min": reverseSwapPairInfo.min,
+            "max": reverseSwapPairInfo.max,
+            "feesHash": reverseSwapPairInfo.feesHash,
+            "feesPercentage": reverseSwapPairInfo.feesPercentage,
+            "feesLockup": reverseSwapPairInfo.feesLockup,
+            "feesClaim": reverseSwapPairInfo.feesClaim
+        ]
+    }
+
+    static func dictionaryOf(reverseSwapInfo: ReverseSwapInfo) -> [String: Any?] {
```

Comment:
Can be `[String: Any]` as there are no optional values

Reply:
Done